python/ql/src/semmle/python/objects/ObjectAPI.qll

```diff
@@ -619,7 +641,20 @@ class BuiltinFunctionValue extends FunctionValue {
 
     override int minParameters() { none() }
 
-    override int maxParameters() { none() }
+    override string descriptiveString() { result = "builtin-function " + this.getName() }
+
+    override int minParameters() {
```

Comment:
All the tests are currently failing due to this double occurrence of the `minParameters` predicate.